src/parse.c

```diff
@@ -1437,31 +1442,43 @@ handle_wifi_access_points(NetplanParser* npp, yaml_node_t* node, const char* key
                 continue;
         }
 
+        ssid = scalar(key);
+
+        /* Skip access-points that already exist in the netdef */
+        if (npp->current.netdef->access_points && g_hash_table_contains(npp->current.netdef->access_points, ssid))
+            continue;
```

Comment:
thought: This ignores the definition of an access-point/SSID if it was already defined in a previous YAML file, AFAIU.

Usually we allow merging of any settings in our schema, where settings from "higher" files (sorting later lexicographically) will win over settings of "lower" files. Here we seem to ignore the "higher" SSID, keeping the "lower" one around. Would it make sense to clear the lower SSID instead and put the new one in place?

Reply:
yeah that might be a bit better. One would expect a behavior like that from Netplan I guess. I'll give it a try...

Reply:
I implemented that in a new commit. Let me know if that's what you had in mind.